topnav/media-menu.js

```diff
@@ -0,0 +1,127 @@
+import { LitElement, html, css } from 'lit-element';
+
+import './media-slider'
+
+class MediaMenu extends LitElement {
+
+  static get properties() {
+    return {
+      mediaMenuOpen: { type: Boolean },
+      mediaMenuAnimate: { type: Boolean },
+      mediaSliderOpen: { type: Boolean },
+      mediaSliderAnimate: { type: Boolean }
+    };
+  }
+
+  constructor() {
+    super();
+    this.mediaSliderOpen = false;
+    this.mediaSliderAnimate = false;
+  }
+
+  mediaSlider() {
+    // Only keep side menu open until menu closed
+    // Shift menu downwards as per button clicked
+    this.mediaSliderAnimate = true;
+    this.mediaSliderOpen = !this.mediaSliderOpen;
+  }
+
+  render() {
+    const mediaMenuClass = this.mediaMenuOpen ? 'media-menu open slide-in' : this.mediaMenuAnimate ? 'media-menu slide-out' : 'media-menu';
+    const mediaMenuHidden = this.mediaMenuOpen ? 'false' : 'true';
+    const mediaMenuExpanded = this.mediaMenuOpen ? 'true' : 'false';
+    return html`
+    <nav
+      class="${mediaMenuClass}"
+      aria-hidden="${mediaMenuHidden}"
+      aria-expanded="${mediaMenuExpanded}"
+    >
+      <!-- Include icon and name inline in a button-->
+      <div class="grid">
+        <button tabindex="-1" @click="${this.mediaSlider}"><div><img src="assets/img/ia-waybackmachine-999.svg"></div></button>
+        <button><div>Wayback Machine</div></button>
```

Comment:
Question: do we need that internal div?

Reply:
This question is for all button items

Reply:
Currently, the divs are being used to create a css grid to align the text and icons